torch/_ops.py

```diff
@@ -853,6 +853,20 @@ def has_composite_implicit_kernel(op):
         return False
     return True
 
+# Private API that returns a dict containing all ATen operators
+# that have CompositeImplicitAutograd decompositions registered in the dispatcher.
+def _get_composite_implicit_autograd_decomps() -> Dict[OpOverload, Callable]:
+    import re
+    # Get all C++ CompositeImplicitAutograd decomps.
+    op_names = [re.split(r'::|\.', x) for x in torch._C._dispatch_get_registrations_for_dispatch_key("CompositeImplicitAutograd")]
```

Comment:
sigh lol

Reply:
You have two occurrences of this, can we at least have a helper function lol

Reply:
Agreed haha